Admin/AbstractAdmin.php

```diff
@@ -2845,6 +2852,22 @@ public function getDashboardActions()
     /**
      * {@inheritdoc}
      */
+    public function getSearchResultLink($object)
```

Comment:
Yes. When in doubt, make it final. Making it not final later will not be a BC-break, but the opposite will.